i18n/en-US.properties

```diff
@@ -1553,15 +1553,17 @@ boxui.unifiedShare.collaborators.expirationTooltipClickableText = Access expires
 # Text to show when the number of contact email addresses displayed on a tooltip exceeds the maximum amount that can be displayed
 boxui.unifiedShare.contactEmailsTooltipText = {emails}, and {remainingEmailsCount} more
 # Text for the notice that is displayed when there are collaboration restrictions that apply to one or more of the selected contacts
-boxui.unifiedShare.contactRestrictionNotice = Invitations cannot be sent to {count, plural, one {{count} person} other {{count} people}} because external collaboration is restricted due to the applied security policy.
+boxui.unifiedShare.contactRestrictionNotice = {count, plural, one {{count} invitation} other {{count} invitations}} cannot be sent because external collaboration is restricted due to the applied security policy.
```

Comment:
Replacing all mentions of "people" now that the message can refer to both individual people and groups.